client/src/components/header/header.vue

```diff
@@ -3,10 +3,27 @@ import { useChannelStore } from '../../stores/channel';
 import { default as Button } from '../generic-button/generic-button.vue';
 import GameInfo from '../game-info/game-info.vue';
 import { resetApp } from '../../main';
-import { computed } from 'vue';
+import { computed, onUpdated, reactive, ref } from 'vue';
+import {
+  IS_USING_LOCAL_NODE,
+  NODE_URL,
+} from '../../utils/sdk-service/sdk-service';
 
 const channelStore = useChannelStore();
-const repoURL = 'https://github.com/aeternity/state-channel-demo';
+const repoUrl = 'https://github.com/aeternity/state-channel-demo';
+const explorerUrl = ref('');
+
+const channelWrapper = reactive({
+  channelStore,
+});
+
+onUpdated(() => {
+  if (IS_USING_LOCAL_NODE) {
+    explorerUrl.value = `${NODE_URL}/v2/channels/${channelWrapper.channelStore.channel?.channelId}`;
+  } else if (channelWrapper.channelStore.channel?.channelConfig?.responderId) {
+    explorerUrl.value = `https://testnet.aenalytics.org/accounts/${channelWrapper.channelStore.channel?.channelConfig?.responderId}`;
+  }
+});
```

Comment:
```suggestion
const explorerUrl = computed(() => {
  console.log('hello');
  if (IS_USING_LOCAL_NODE) {
    return `${NODE_URL}/v2/channels/${channelWrapper.channelStore.channel?.channelId}`;
  } else
    return `https://testnet.aenalytics.org/accounts/${channelWrapper.channelStore.channel?.channelConfig?.responderId}`;
});
```

On this video, the proposed method is used (with a `console.log('hello')` inside). 
You can see it is called only once.
[Screencast from 02-09-2022 12:02:27 ΜΜ.webm](https://user-images.githubusercontent.com/45261205/188105308-1ff008eb-279a-45b3-aca9-b672b9a84502.webm)


Here, you can see that the `explorerUrl` will be updated on each component update
[Screencast from 02-09-2022 12:03:51 ΜΜ.webm](https://user-images.githubusercontent.com/45261205/188105318-7480ff23-e00c-48fd-9343-dc3430c136a9.webm)